src/js/samples/components/Detail/__tests__/General.test.jsx

```diff
@@ -1,22 +1,21 @@
-import { shallow } from "enzyme";
+import { createBrowserHistory } from "history";
 import React from "react";
 import { beforeEach, describe, expect, it } from "vitest";
+import { createFakeSample } from "../../../../../tests/fake/samples";
+import { renderWithRouter } from "../../../../../tests/setupTests";
 import { mapStateToProps, SampleDetailGeneral } from "../General";
+
 describe("<SampleDetailGeneral />", () => {
     let props;
+    let history;
 
     beforeEach(() => {
         props = {
             count: 235,
             encoding: "Sanger / Illumina 2.1",
             gc: "42.3%",
-            host: "Malus domestica",
-            isolate: "Isolate Foo",
             lengthRange: "41 - 76",
-            locale: "Bar",
-            name: "Foo",
-            notes: "Example notes",
-            paired: false,
+            sample: {},
```

Comment:
I redefine the prop in the tests that use it using `createFakeSample`. Didn't declare it in the props because the tests that use it need different properties so I just defined it in the test directly.